html/browsers/windows/noreferrer--cross-origin-window-name-manual.sub.html

```diff
@@ -1,3 +1,3 @@
 <meta charset=utf-8>
 <p>Follow this link to open a new browsing context and then confirm it says "idonteven":
-<a rel=noreferrer target=idonteven href="//天気の良い日.{{location[host]}}/html/browsers/windows/support-window-name-echo.html">link</a>.
+<a rel=noreferrer target=idonteven href="//天気の良い日.{{location[host]}}/html/browsers/windows/support-noreferrer--cross-origin-window-name.html">link</a>.
```

Comment:
Another thing that might be worth considering is putting the support resources into a subdirectory (generally named `resources/`). That might also make it easier to keep track of things. (Note that support resources are shared at times so strict matching isn't always feasible.)

I guess I don't care strongly about the double hyphen thing, it just seems a little weird since I haven't seen it used before.